src/components/Registration/Waivers.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import Button from 'react-bootstrap/Button';
+
+import LiabilityWaiver from 'static/waivers/exploretech.la 2021 Liability Waiver.pdf';
+import MultimediaRelease from 'static/waivers/exploretech.la 2021 Multimedia Release Form.pdf';
+import StudentVerification from 'static/waivers/exploretech.la 2021 Student Verification Form.pdf';
```

Comment:
Could we please rename the `waivers` directory to `pdf` so we can put other PDFs there in the future? So it would become `static/pdf/...`. If you want to do more organization, you could even create a `waivers` directory under `pdf`.

Reply:
Changing to `pdf` for now, may add a `waivers` directory in the future